CHANGELOG.md

```diff
@@ -3,6 +3,80 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.0.0-alpha.9
+
+<!-- generated comparing v6.0.0-alpha.8..next -->
+
+_Nov 24, 2022_
+
+We'd like to offer a big thanks to the 14 contributors who made this release possible. Here are some highlights ✨:
+
+- 💅 Add support for `theme.vars` in the pickers and the DataGrid (#6784, #6778) @alexfauquette
```

Comment:
@LukasTy @alexfauquette @joserodolfofreitas 
I don't know how to present that big of a change :grimacing: 

```suggestion
- 🎁 Introduce the v6 pickers, built on top of the field components ([DatePicker](https://next.mui.com/x/react-date-pickers/date-picker/), ([TimePicker](https://next.mui.com/x/react-date-pickers/time-picker/), ([DateTimePicker](https://next.mui.com/x/react-date-pickers/date-time-picker/), ([DateRangePicker](https://next.mui.com/x/react-date-pickers/date-range-picker/)

  The old components will be removed at the end of the v6 beta.

- 💅 Add support for `theme.vars` in the pickers and the DataGrid (#6784, #6778) @alexfauquette
```

Reply:
To me it sounds like a really good short summary. 🤔 🙈 
We don't want to make the changelog crowded...

Just maybe add `legacy` name to make more sense with how the documentation treats "old" components.
```diff
-The old components will be removed at the end of the v6 beta.
+The old (legacy) components will be removed at the end of the v6 beta.
```

Reply:
We can add a link to the blogpost when released

Reply:
The introduction sounds good 👍. Maybe bullet points instead of commas to list the next components. To give them more space/visibility

Reply:
Ok, I forget we had other things to highlight 👍